### DISCUSSION TITLE
Mobile apps

### DISCUSSION THREAD

**Discussion by kenchoong** [dailydotdev/daily#490] [Feature Requests]
Will love to use this daily.dev available in Android and iOS, is it exists right now? 

Or having any plans to make this happen?

**Comment by nimrodkra** [comment]
Hey @kenchoong, thanks for taking the time to write here :) 

For Android, we have our PWA available through Google Play: https://play.google.com/store/apps/details?id=dev.daily
For iOS, we have this temporary solution that allows you to [add a website icon to your iPhone/iPad Home Screen](https://docs.daily.dev/docs/getting-started/pwa#how-can-i-add-dailydev-to-my-mobile-home-screen-on-ios) 

We do plan to create native apps in the future, but we can't commit to a schedule at the moment.

**Reply by kenchoong** [reply]
Tq for your guidance . I just miss out the docs
